### ISSUE TITLE
Set maskSensitiveHeaderData default to true

### ISSUE THREAD

**Issue by daern91** [commercetools/nodejs#525]
### Description
We never set a default value for this option in [sdkMiddlewareHttp](https://github.com/commercetools/nodejs/blob/d6e9751364c2cf83a02ce0070be1c18d634f172e/docs/sdk/api/sdkMiddlewareHttp.md) and it is not covered in the documentation. This should be considered and implemented.

Broken out from #518. 

### Expected Behavior
`maskSensitiveHeaderData` should default to `true`

### Current Behavior
There is no default specified in documentation or code.

**Comment by tdeekens** [comment]
It makes sense to me to default it. Can we get any information if anyone actually uses the data to consider how breaking the change is? I think we should also briefly get @emmenko opinion on this.

**Comment by emmenko** [comment]
Makes sense yes. We could maybe bump a major version when releasing, to indicate that this might be a breaking change.

**Comment by wizzy25** [comment]
Personally, I think this is a great idea. It shouldn't be a problem for the node packages depending on it. I however don't know how significant the breakage will be for other users of the SDK.
So yeah, we should go for it 👍🏾
